extension/llm/runner/CMakeLists.txt

```diff
@@ -44,18 +44,23 @@ target_include_directories(
 add_library(extension_llm_runner STATIC ${_extension_llm_runner__srcs})
 
 # add tokenizers
-add_subdirectory(
-  ${EXECUTORCH_ROOT}/extension/llm/tokenizers
-  ${CMAKE_CURRENT_BINARY_DIR}/../../../extension/llm/tokenizers
+ExternalProject_Add(
```

Comment:
So, if i understand correctly, ExternalProject_Add creates a separate build process and as a result cmake variables aren't shared. 

If you're cross-compiling, don't you wanna propagate toolchain settings and compiler flags? 

I think FetchContent propagates compiler flags and toolchain setting. Looks like FetchContent_Declare and FetchContent_MakeAvailable will do the job, and it happens during config time, not build time. 

Also, fwiw, looks like XNNPACK is fixing this issue (https://github.com/google/XNNPACK/issues/8488) with this PR (https://github.com/google/XNNPACK/pull/8552)

Reply:
![alt text](https://media4.giphy.com/media/v1.Y2lkPTc5MGI3NjExZmE0Y2sxN28xZHN0OXpxZ2duMXFsbzUwcDE1MHF0Ync4dDZqN24ybyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3otPoUjeyRisIDxPhK/giphy.gif)